airplane/The_Spiroc_Lord.lua

```diff
@@ -1,7 +1,7 @@
 function event_death_complete(e)
 	local qglobals = eq.get_qglobals();
 	
-	if(qglobals["sirran"] ~= "5" and eq.get_entity_list():IsMobSpawnedByNpcTypeID(71009) == false) then
+	if qglobals["sirran"] ~= "5" and eq.get_entity_list():IsMobSpawnedByNpcTypeID(71009) then
```

Comment:
This checked for false before, we’re not checking for false now.

Reply:
Thanks - Resolved